doc/glossary.rst

```diff
@@ -184,8 +186,12 @@
                 store Ceph-specific information about devices and its
                 relationship with OSDs.
 
-	MDS
-		The Ceph metadata software.
+	:ref:`MDS<cephfs_add_remote_mds>`
+                The Ceph metadata server daemon. Also referred to as
+                "ceph-mds". The Ceph metadata server daemon is required to run
+                the CephFS file system. The MDS (the daemon) stores all
+                filesystem metadata in the memory of high-availability Ceph
+                Metadata Servers. 
```

Comment:
This seems self-referential.  

s/(the daemon)//
Suggest truncating after “filesystem metadata”

Reply:
I see your point. I'm going to make that alteration.